manifests/g/GDKSoftware/Duster/1.1.0.2/GDKSoftware.Duster.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GDKSoftware.Duster
+PackageVersion: 1.1.0.2
+PackageLocale: en-US
+Publisher: GDKSoftware
+PackageName: Duster
+License: NLdigital Terms
+ShortDescription: Duster

```

Comment:
```
ShortDescription: Duster
```
Please write a better description for what this application is about.

Reply:
Thanks, we're working on that. Could we revert this pull request for now?